### PR TITLE
Adding a new page - "Permission levels in an organization"

### PR DESCRIPTION
**Pull request by dunjardl**

### CODE REVIEW

**Comment by dunjardl** [comment]
@pyromaniackeca Please review.

File: source/docs/organizations/permission-levels-in-an-organization.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+### 4. Owner
+
+The Owners team is able to:
+
+- Access projects
+- Change settings
+- Add branches
+- Manually trigger builds and deploys
+- Add and remove projects
```
Comment (pyromaniackeca) [inline source/docs/organizations/permission-levels-in-an-organization.md]: Maybe change this line to: `Add and remove projects from the organization account`, so it matches the item above.
People might think they are two different things.

**Comment by pyromaniackeca** [comment]
:+1: It looks great, just add this page to the index and I think it's ok for merging.

File: source/docs/organizations/permission-levels-in-an-organization.md (hunk 1 above)
Comment (dunjardl) [inline source/docs/organizations/permission-levels-in-an-organization.md]: Make sense, I'll update that bullet point.

**Comment by dunjardl** [comment]
:+1: Thanks.

**Comment by dunjardl** [comment]
@pyromaniackeca Please review and merge.

**Pull request merged**